app/services/explorer.js

```diff
@@ -655,44 +657,42 @@ export default Ember.Service.extend({
      * @param {DS.Store} store
      * @return {Ember.RSVP.Promise<BucketList>} Result of the AJAX request
      */
-    getBucketList(cluster, bucketType, store) {
-        console.log('Refreshing buckets for bucketType: %O', bucketType);
-        var clusterId = cluster.get('clusterId');
-        var bucketTypeId = bucketType.get('bucketTypeId');
-        var url = this.apiURL + 'explore/clusters/' + clusterId +
-            '/bucket_types/' + bucketTypeId + '/buckets' ;
-        var explorer = this;
+    getBucketList(cluster, bucketType, store, start=1, rows=100) {
```

Comment:
Alter the default value in rows if you want to test this out locally and want the pagination chunks to be shorter. This value gets set as an attribute on the model, and the component picks it up from there.